phpunit.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
+         bootstrap="vendor/autoload.php"
+         cacheResultFile=".phpunit.cache/test-results"
+         executionOrder="depends,defects"
+         beStrictAboutCoversAnnotation="true"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutTodoAnnotatedTests="true"
+         convertDeprecationsToExceptions="true"
+         failOnRisky="true"
+         failOnWarning="true"
+         verbose="true">
+    <testsuites>
+        <testsuite name="default">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <coverage cacheDirectory=".phpunit.cache/code-coverage"
```

Comment:
Is dit nog nuttig?

Reply:
Ik heb gewoon `vendor/bin/phpunit --generate-configuration` en toen kwam dit eruit. Als jij zegt dat het eruit kan/mag, dan geloof ik dat meteen

Reply:
Wat mij betreft wel, daar doen we verder nooit iets mee. En coverage staat ook "uit" in de Github action? Anyway...weg er mee :slightly_smiling_face:

Reply:
Het is weg